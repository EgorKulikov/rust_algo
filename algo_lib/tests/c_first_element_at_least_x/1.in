5 7
1 3 2 4 6
2 2
2 5
1 2 5
2 4
2 8
1 3 7
2 6
