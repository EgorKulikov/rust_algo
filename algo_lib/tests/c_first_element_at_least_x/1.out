1
4
2
-1
3
